{
  "_annotation": {
    "cve_id": "CVE-2024-6345",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-6345.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:python:setuptools:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "6661BB7C-1684-519E-97CF-5CE974BD9651",
                "versionEndExcluding": "70.0.0",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}